crates/sats/src/ser.rs

```diff
@@ -203,13 +203,17 @@ pub use spacetimedb_bindings_macro::Serialize;
 
 use crate::AlgebraicType;
 
-/// A **data structure** that can be serialized into any data format supported by SATS.
+/// A **data structure** that can be serialized into any data format supported by
```

Comment:
Why is "data structure" emphasized here, and can it not be?

Reply:
tyler